iati/home/models.py

```diff
@@ -25,6 +25,15 @@ class Meta:
         icon = "openquote"
 
 
+class ImageAlignmentChoiceBlock(FieldBlock):
```

Comment:
While you're here can you add some basic docstrings to these classes so the linter will be happy later?